frontend/packages/dev-console/src/components/pipelines/pipeline-overview/PipelineOverview.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { OverviewItem, Status } from '@console/shared';
+import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
+import * as _ from 'lodash';
+import { ListGroup } from 'patternfly-react';
+import { Grid, GridItem } from '@patternfly/react-core';
+import {
+  SidebarSectionHeading,
+  ResourceLink,
+  resourcePath,
+} from '@console/internal/components/utils';
+import { PipelineRunModel, PipelineModel } from '@console/dev-console/src/models';
+import { Link } from 'react-router-dom';
+import { fromNow } from '@console/internal/components/utils/datetime';
+import { pipelineRunFilterReducer } from '../../../utils/pipeline-filter-reducer';
+import TriggerLastRunButton from './TriggerLastRunButton';
+
+const noOfPipelineRunsToshow = 3;
+
+type PipelineRunItemProps = {
+  pipelineRun: K8sResourceKind;
+};
+
+export const PipelineRunItem: React.FC<PipelineRunItemProps> = ({ pipelineRun }) => {
+  const {
+    metadata: { name, namespace, creationTimestamp },
+    status: { completionTimestamp, startTimestamp },
+  } = pipelineRun;
+  const path = resourcePath(referenceForModel(PipelineRunModel), name, namespace);
+  const lastUpdated = completionTimestamp || startTimestamp || creationTimestamp;
+  return (
+    <li className="list-group-item">
+      <Grid gutter="md">
```

Comment:
I feel this `Grid` is unbalanced as well, but it aligns with `Pods` so we'll have to keep this. I think we could definitely improve the usage of space here, there is unnecessary whitespace gaps.